spec/views/user_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  before(:each) do
+    @user = User.create(
+      name: 'Tom',
+      photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+      bio: 'Teacher from Mexico.'
+    )
+  end
+  describe 'GET /index' do
+    before(:example) { get users_path }
+
+    it 'is a success' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'Should include the template text' do
+      expect(response.body).to include('Tom')
+    end
+
+    it 'renders index template' do
+      expect(response).to render_template(:index)
+    end
+  end
+
+  describe 'GET /show' do
+    before(:example) do
+      get "/users/#{@user.id}"
+    end
+
+    it 'is a success' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'Should include the template text' do
+      expect(response.body).to include('Teacher from Mexico.')
+    end
+    it 'renders show template' do
+      expect(response).to render_template(:show)
+    end
+  end
```

Comment:
- [ ] Kindly add the tests missing on this file

![image](https://github.com/berylcl/Rails-Blog-app/assets/93630700/556fb7c0-68eb-48be-bce8-fb605bedbf22)